### ISSUE TITLE
Time delta calculation bug when both start and end times are after hours

### ISSUE THREAD

**Issue by brox** [seatgeek/businesstime#13]
This piece of code returns the wrong time delta:

```
>>> bt=BusinessTime()
>>> bt.businesstimedelta(datetime(2016, 6, 17, 18), datetime(2016, 6, 20, 18))
datetime.timedelta(0)
```

The start time is after hours on a Friday at 6pm, while the end time is after hours on the following Monday at 6pm. The correct answer should be `datetime.timedelta(1)`, right?

**Comment by flc** [comment]
assuming `bt.business_hours` is `(datetime.time(9, 0), datetime.time(17, 0))` and `bt.weekends` is `(5, 6) `shouldn't this be equal to `bt.businesstimedelta(datetime.datetime(2016, 6, 20, 9, 0), datetime.datetime(2016, 6, 20, 17, 0))` ?
which is:
`datetime.timedelta(0, 28800)`

**Comment by josegonzalez** [comment]
Closed by #22
